bazel/boringssl_static.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/BUILD b/BUILD
+index d7c731bf6..315cdeca0 100644
+--- a/BUILD
++++ b/BUILD
+@@ -88,6 +88,7 @@ boringssl_copts = select({
+     ":windows_x86_64": [
+         "-DWIN32_LEAN_AND_MEAN",
+         "-DOPENSSL_NO_ASM",
++        "-DBORINGSSL_IMPLEMENTATION",
+     ],
+     "//conditions:default": ["-DOPENSSL_NO_ASM"],
+ })
+@@ -141,6 +142,7 @@ cc_library(
+         ":windows_x86_64": ["-defaultlib:advapi32.lib"],
+         "//conditions:default": ["-lpthread"],
+     }),
++    linkstatic = True,
```

Comment:
I think you can just pass `--dynamic_mode=off` in windows .bazelrc instead of patching build targets.

Reply:
Confirmed and addressed in latest patch

Reply:
Apparently our confirmation wasn't detailed enough. With the following .bazelrc;

build --features=fully_static_link
build --features=static_link_msvcrt
build --dynamic_mode=off

The boringssl BAZEL file will continue to try to copy an export library which doesn't exist for a static .lib'rary, even though it didn't attempt to compile dynamic objects or a dynamic library.

Only the linkstatic flag appears to inhibit the copying of non-existent artifact.

Calling this out here for @davidben @agl to attend to in boringssl's bazel build schema.